### PR TITLE
Ignore stroking when stroke-width is less then or equal zero.

### LINKED ISSUE #137: Panics when stroke width is std::f32::MIN
```rust
use raqote::*;

fn main() {
    let mut dt = DrawTarget::new(400, 400);

    let mut pb = PathBuilder::new();
    pb.move_to(200., 200.);
    pb.line_to(300., 300.);
    pb.line_to(200., 300.);

    let path = pb.finish();
    dt.stroke(
        &path,
        &Source::Solid(SolidSource::from_unpremultiplied_argb(0xFF, 0, 0x80, 0)),
        &StrokeStyle {
            width: std::f32::MIN, // <--
            ..StrokeStyle::default()
        },
        &DrawOptions::new(),
    );

    dt.write_png("out.png").unwrap();
}
```

```
thread 'main' panicked at 'attempt to subtract with overflow', src/rasterizer.rs:361:26
```

Somewhat related to #135

### PR DESCRIPTION
**Pull request by RazrFalcon**
Closes #137

### CODE REVIEW

File: src/stroke.rs [hunk 1]
```diff
@@ -275,8 +275,13 @@ fn join_line(
 }
 
 pub fn stroke_to_path(path: &Path, style: &StrokeStyle) -> Path {
-    let mut cur_pt = None;
     let mut stroked_path = PathBuilder::new();
+
+    if style.width < 0.0 {
```
Comment (jrmuizel) [inline src/stroke.rs:280]: The commit message says `<=` but the code says `<`

**Comment by RazrFalcon** [comment]
Fixed.

**Comment by jrmuizel** [comment]
Why not just use `<=` ?

**Comment by RazrFalcon** [comment]
Floats comparison is tricky, therefore I'm using this style. But looks like `<=` should also work.

**Comment by RazrFalcon** [comment]
Updated and added a new test.

**Pull request merged**